diff --git a/my-app/src/App.jsx b/my-app/src/App.jsx
index fdbc2f7..7495fef 100644
--- a/my-app/src/App.jsx
+++ b/my-app/src/App.jsx
@@ -23,7 +23,7 @@ class App extends Component {
 			<div className="App">
 				<Menu
 					activeSection={this.state.activeSection}
-					changeSection={(section) => this.setState({ section })}
+				    changeSection={(section) => this.setState({ activeSection: section })}
 				/>
 
 				<Startseite />
diff --git a/my-app/src/Components/Menu.jsx b/my-app/src/Components/Menu.jsx
index 2f0759e..c3cb960 100644
--- a/my-app/src/Components/Menu.jsx
+++ b/my-app/src/Components/Menu.jsx
@@ -1,87 +1,39 @@
-import './Menu.css'
-import React, { Component } from 'react'
+import "./Menu.css";
+import React, { Component } from "react";
 
 class Menu extends Component {
-	constructor(props) {
-		super(props)
-		this.state = {
-			didScroll: false,
-			// activeSection: '',
-		}
-	}
-	componentDidMount() {
-		window.addEventListener('scroll', () => {
-			this.setState({
-				didScroll: true,
-			})
-		})
-
-		setInterval(() => {
-			if (this.state.didScroll) {
-				this.setState({
-					didScroll: false,
-				})
-				let activeSection
-
-				const scroll = window.scrollY
-
-				const screenHeight = window.innerHeight
-
-				//Same that all the if else statements
-				// switch (true) {
-				//   case scroll >= 0 && scroll <= screenHeight:
-				//     activeSection = 1;
-				//     break;
-				//   case scroll >= screenHeight && scroll <= screenHeight*2:
-				//     activeSection = 2;
-				//     break;
-				//   case scroll >= screenHeight*2 && scroll <= screenHeight*3:
-				//     activeSection = 3;
-				//     break;
-				//   case scroll >= 2970:
-				//     activeSection = 4;
-				//     break;
-				//   //   default:
-				//   //scroll<=590
-				// }
-
-				activeSection = Math.floor(scroll / screenHeight) + 1
-
-				this.setState({
-					activeSection,
-				})
-			}
-		}, 50)
-	}
-	render() {
-		'let activeSection = this.'
-
-		return (
-			<nav>
-				<ul id="navigation">
-					<li className={activeSection == 'start' ? 'active' : ''}>
-						<a href="#startseite" onClick={() => this.props.activeSection}>
-							Startseite
-						</a>
-					</li>
-					<li className={activeSection == 'formular' ? 'active' : ''}>
-						<a href="#formular">Projektformular</a>
-					</li>
-					<li className={activeSection == 'test' ? 'active' : ''}>
-						<a href="#test">Test</a>
-					</li>
-					{/* <li className={activeSection == 4 ? 'active' : ''}>
-						<a href="#part_4">Prinzipien</a>
-					</li> */}
-					<li className={activeSection == 'chat' ? 'active' : ''}>
-						<a href="#chat">Buddy Chat</a>
-					</li>
-					{/* <li className={activeSection == 6 ? 'active' : ''}>
-						<a href="#part_6">Ende</a>
-					</li> */}
-				</ul>
-			</nav>
-		)
-	}
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { activeSection, changeSection } = this.props;
+
+    return (
+      <nav>
+        <ul id="navigation">
+          <li className={activeSection == "start" ? "active" : ""}>
+            <a href="#startseite" onClick={() => changeSection("start")}>
+              Startseite
+            </a>
+          </li>
+          <li className={activeSection == "formular" ? "active" : ""}>
+            <a href="#formular" onClick={() => changeSection("formular")}>
+              Projektformular
+            </a>
+          </li>
+          <li className={activeSection == "test" ? "active" : ""}>
+            <a href="#test" onClick={() => changeSection("test")}>
+              Test
+            </a>
+          </li>
+          <li className={activeSection == "chat" ? "active" : ""}>
+            <a href="#chat" onClick={() => changeSection("chat")}>
+              Buddy Chat
+            </a>
+          </li>
+        </ul>
+      </nav>
+    );
+  }
 }
-export default Menu
+export default Menu;
diff --git a/my-app/src/Pages/Austausch.jsx b/my-app/src/Pages/Austausch.jsx
index 492d7e6..7878251 100644
--- a/my-app/src/Pages/Austausch.jsx
+++ b/my-app/src/Pages/Austausch.jsx
@@ -14,7 +14,7 @@ class Austausch extends Component {
 	render() {
 		return (
 			<>
-				<section id="part_5" tabIndex="5">
+				<section id="chat" tabIndex="5">
 					<Match language={this.props.language} />
 					<Chat />
 					{/* <h2>Test</h2> */}
